src/gridcoin/voting/payloads.h

```diff
@@ -56,8 +63,15 @@ class PollPayload : public IContractPayload
     //! \param claim Testifies that the poll author owns the required balance.
     //!
     PollPayload(Poll poll, PollEligibilityClaim claim)
-        : PollPayload(CURRENT_VERSION, std::move(poll), std::move(claim))
     {
+        m_version = CURRENT_VERSION;
+
+        if (!IsPollV3Enabled(nBestHeight)) {
```

Comment:
I am not changing the constructor for PollPayload(Poll poll, PollEligibilityClaim claim) because it causes breaking in the whole MakeContract formalism in the contract interface.